### ISSUE TITLE
BIG scare during installation, but resolved phew!!!  = Suggested warning for installation docs.

### ISSUE THREAD

**Issue by deryb** [jenv/jenv#308]
Am a fairly beginner linux user (actually on WSL, Windows Subsystem for Linux).

Just had a BIG scare yesterday evening, pulled my hair for a few hours and thought I had lost my custom bash profile. Plus, this made me realize I don't have backups of it (my bad) !! Here's the cascade of events:

The install doc says, for linux:
```
$ echo 'export PATH="$HOME/.jenv/bin:$PATH"' >> ~/.bash_profile
$ echo 'eval "$(jenv init -)"' >> ~/.bash_profile
```

but Ubuntu uses `~/.profile` instead. Unless there's a .bash_profile!  So your little two lines of code, while just appending which should do no harm, actually took over my bash config because ubuntu uses .bash_profile if it finds one. Meheheheh (evil laugh).

So I reloaded my terminal a few times, wondered why it overwrote config, then tried to recover whatever backups I might've had (it's WSL, so Window's file backup does nothing on it!). Eventually just gave up and went to sleep, zzzz.

Then today found the bit in docs about Ubuntu using .profile normally. Checked my dir and tadam! My old custom .profile is still there and intact, so all I have to do is disable the created .bash_profile (by your install command) and everything is back to how it was! PHEW!!

Of course, a more advanced Linux user would've probably figure this, but I suggest maybe you should add just a little warning text, or perhaps word it differently:  IF you have a .bash_profile, add these two lines. If not, add them to your .profile instead. And of course: Kids, make backups before editing your precious config files!

Thanks! Now I'm eager to try out this tool, in a MUCH BETTER MOOD now...  :-)

**Comment by voronaam** [comment]
It sounds like there might be some interest in creating a jenv dpkg package for Ubuntu, such that manual installation instructions are only for the experienced users, while new Linux users might rely on something like `apt-get install jenv`.

If there was a package in Ubuntu repos, would've you tried it before the manual installation steps?

**Comment by deryb** [comment]
Yes, definitely! Being fairly novice at linux & bash, I'd feel more confident using apt-get than manually modifying scripts. Thanks, sorry for the late reply it's been busy lately here!
